pkg/cfn/builder/cluster.go

```diff
@@ -204,6 +214,17 @@ func (c *ClusterResourceSet) addResourcesForControlPlane(subnetDetails *subnetDe
 	}
 }
 
+func makeCFNTags(clusterConfig *api.ClusterConfig) []gfncfn.Tag {
```

Comment:
I swear we have one of these somewhere...

Reply:
ah yeah, we have `makeTags` but that's doing something else as well. Wish we could consolidate them, but not now. :)